package.json

```diff
@@ -15,11 +15,13 @@
     "test": "vitest",
     "test:update-snapshots": "vitest run -u",
     "test:coverage": "vitest --coverage",
-    "docs:dev": "docsify serve docs",
     "premigrate": "run-s build:bin",
     "migrate": "node bin/node-pg-migrate.js",
     "prepublishOnly": "pnpm run clean && pnpm install && pnpm run build",
-    "preflight": "pnpm install && run-s format build lint test:update-snapshots ts-check"
+    "preflight": "pnpm install && run-s format build lint test:update-snapshots ts-check",
+    "docs:dev": "vitepress dev docs",
+    "docs:build": "vitepress build docs",
+    "docs:preview": "vitepress preview docs"
```

Comment:
Please move them to where the docs where previously (~line 18, above premigrate)